modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts

```diff
@@ -22,6 +22,7 @@ const english: Record<string, string> = {
   'colorcustom.rgb.hex.description': 'Hex color code',
   'colorcustom.rgb.range': 'Range 0 to 255',
   'colorcustom.rgb.invalid': 'Numbers only, 0 to 255',
+  'colorcustom.rgb.invalidHex': 'Hexadecimal only, 000000 to FFFFFF',
```

Comment:
This should be also added to the `.pot` file

Reply:
That's not part of the dedicated translation issue?

Reply:
Hmm.. Looks like [`Numbers only` is also missing in the translation](https://github.com/tinymce/tinymce-premium/blob/main/tools/language-packs/src/text/pot/core/tinymce_core.pot#L1407).

Reply:
Yes, they were never added to any translation thing, as that was a separate issue.

Reply:
Hmm, ok. I've always added the new strings to `pot` files. With context to not have someone go through all of them again adding contexts. 
Do you have some additional ticket to add both errors to `pot` files?